linode/manager
# Gravatar Clean Up

## Description 📝

- Cleans up Gravatar Logic and Usage
- We were doing way too much heavy lifting when using gravatars
  - We used axios to see if a gravatar existed but this is super redundant. It will just 404 if it does not exist.
- Use the MUI `<Avatar />` to handle fallback to our default icon

## Preview 📷

- Nothing should appear differently. This is a "under the hood" change

## How to test 🧪

- Test Gravatars throughout Cloud Manager
  - View `/account/users`
  - Make sure your gravatar shows in top navigation menu
  - Make sure gravatars show correctly in the events feeds (`/events`)
  - Make sure gravatars show correctly for support tickets and support ticket replies
